unstructured/ingest/cli/cmds/astra.py

```diff
@@ -36,13 +36,6 @@ def get_cli_options() -> t.List[click.Option]:
                 "Note that the collection name must only include letters, "
                 "numbers, and underscores.",
             ),
-            click.Option(
-                ["--embedding-dimension"],
-                required=True,
-                default=384,
-                type=int,
-                help="The dimensionality of the embeddings",
-            ),
             click.Option(
                 ["--namespace"],
                 required=False,
```

Comment:
`--requested-indexing-policy` might also be a destination exclusive arg

Reply:
Yes. That sounds true. But lets check with @erichare

Reply:
Right... these apply to the destination collection in Astra DB for sure. Both the indexing policy and the embedding dimension. So that's a very good point, these should really only apply for that. The source connector assumes the collection already exists and so these parameters are not needed.

Good catch. Thanks!